### PR TITLE
feat: Add storybook

### PR DESCRIPTION
**Pull request by simchaNielsen**

### CODE REVIEW

File: jest.config.base.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ module.exports = {
   testEnvironment: 'jsdom',
   timers: 'real',
   verbose: true,
+  snapshotSerializers: ["@emotion/jest/serializer"],
```
Comment (Talzen) [inline jest.config.base.js:15]: why is emotion needed for the snapshot ?

File: packages/storybook/.storybook/main.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  },
+  "stories": [
+    "../stories/**/*.stories.mdx",
+    "../stories/**/*.stories.@(js|jsx|ts|tsx)"
+  ],
+  "addons": [
+    "@storybook/addon-links",
+    "@storybook/addon-essentials"
+  ]
+}
```
Comment (Talzen) [inline packages/storybook/.storybook/main.js:19]: new line should added

File: packages/storybook/.storybook/preview.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+export const parameters = {
+  actions: { argTypesRegex: "^on[A-Z].*", layout: 'centered' },
+}
```
Comment (Talzen) [inline packages/storybook/.storybook/preview.js:4]: new line should added

File: jest.config.base.js (hunk 1 above)
Comment (simchaNielsen) [reply jest.config.base.js:15]: 2 reasons:
- Every time name of classes is overriden
- Sometimes we use <Grid> where templateColumns css property is generated dynamically so I need also know that they generated correctly

File: packages/storybook/.storybook/main.js (hunk 2 above)
Comment (simchaNielsen) [reply packages/storybook/.storybook/main.js:19]: in future PR 👍

File: packages/storybook/.storybook/preview.js (hunk 3 above)
Comment (simchaNielsen) [reply packages/storybook/.storybook/preview.js:4]: 👍 same

**Pull request closed without merging**